Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingHelloWorld/D3D12RaytracingHelloWorld.cpp

```diff
@@ -193,15 +193,13 @@ void D3D12RaytracingHelloWorld::CreateLocalRootSignatureSubobjects(CD3D12_STATE_
         rootSignatureAssociation->AddExport(c_raygenShaderName);
     }
 #if USE_NON_NULL_LOCAL_ROOT_SIG 
-    // Empty local root signature to be used in a miss shader and a hit group.
+    // Empty local root signature is set as the default shader association.
```

Comment:
Let's leave it being explicit in the Hello World one for demonstration purposes

Reply:
And please add: 
    // In this sample, this could be omitted for convenience and use the default shader association instead, since <why>.

Reply:
Note: the end of the comment is TBD/not finished.